src/main/webapp/gameDialogue.js

```diff
@@ -95,12 +101,14 @@ function modifyIfFinalStage() {
 
 //play button functions
 function playmusic() {
-  //TODO: change the play and pause to toggle on and off icons
-  if(document.getElementById('player-button').innerText == "PAUSE"){
-    document.getElementById('player-button').innerText = "PLAY";
-  }
-  else{
-    document.getElementById('player-button').innerText = "PAUSE";
+  if(isPlayingmusic) {
+    const promotionMEME = document.getElementById(MUSIC_CONTAINER);
+    promotionMEME.innerHTML = UNMUTE_ICON;
```

Comment:
why is this named promotionMEME?

Reply:
+1

Also, it looks like `const promotionMeme = document.getElementById(MUSIC_CONTAINER)` could be set outside of the `if/else` block